gradle.properties

```diff
@@ -37,5 +37,6 @@ POM_DEVELOPER_NAME=Ronaldo Pace
 # http://stackoverflow.com/questions/8587096/how-do-you-debug-java-annotation-processors-using-intellij
 # https://turbomanage.wordpress.com/2014/06/09/debug-an-annotation-processor-with-intellij-and-gradle/
 # https://turbomanage.files.wordpress.com/2014/06/remote-debug.png
-org.gradle.daemon=true
-org.gradle.jvmargs=-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=5005
\ No newline at end of file
+# Uncomment for debugging
```

Comment:
no. I don't care.. fuck ur mac. it works on linux and that's what allows to break point